src/edu/wright/cs/raiderplanner/controller/SettingsController.java

```diff
@@ -0,0 +1,412 @@
+	//private DropShadow notifShadow = new DropShadow(screenAverage * 0.02, 0, 0.009, Color.BLACK);
+	private DropShadow moduleDefaultShadow = new DropShadow(screenAverage * 0.005, 0, 0,
+			Color.BLACK);
+	private DropShadow moduleHoverShadow = new DropShadow(screenAverage * 0.02, 0, 0, Color.BLACK);
+	private InnerShadow modulePressedShadow = new InnerShadow(screenAverage * 0.017, 0, 0,
+			Color.BLACK);
+
+	// Labels:
+	private Label welcome;
+	@FXML
+	private Label title;
+
+	// Buttons:
+	@FXML
+	private Button openMenu;
+	@FXML
+	private Button generalBtn;
+	@FXML
+	private Button accountBtn;
+	@FXML
+	private Button themeBtn;
+	@FXML
+	private Button notificationsBtn;
+	@FXML
+	private Button aboutBtn;
+	@FXML
+	private Button closeDrawer;
+
+	// Panes:
+	@FXML
+	private AnchorPane navList;
+	@FXML
+	private GridPane mainContent;
+	@FXML
+	private HBox topBox;
+	@FXML
+
+	//chat variables
+	private final BorderPane mainPane = new BorderPane();
+
+	/**
+	 * Sets this.current to equal passed variable and calls this.main().
+	 */
+	public void main(Window wind) {
+		this.current = wind;
+		this.main();
+	}
+
+	/**
+	 * Main method containing switch statements.
+	 * Switch statements detect what state the app is in.
+	 */
+	public void main() {
+		if (isNavOpen) {
+			openMenu.fire();
+		}
+
+		initialLoad = false;
+
+		switch (this.current) {
+		case ABOUT: {
+			loadAbout();
+			break;
+		}
+		case GENERAL: {
+			this.loadGeneral();
+			break;
+		}
+		case ACCOUNT: {
+			this.loadAccount();
+			break;
+		}
+		case THEME: {
+			this.loadTheme();
+			break;
+		}
+		case NOTIFICATIONS: {
+			this.loadNotification();
+			break;
+		}
+		default:
+			break;
+		}
+	}
+
+	/**
+	 * Display the About menu.
+	 */
+	public void loadAbout() {
+		// set ToolTips
+		this.mainContent.getChildren().remove(1, this.mainContent.getChildren().size());
+		this.topBox.getChildren().clear();
+		this.title.setText("About");
+
+		//Text Labels:
+		Label appName = new Label("RaiderPlanner");
+		appName.setFont(Font.font("Ariel", FontWeight.BOLD , 42));
+
+		Label versionNo = new Label("Version 1.0.1\nCopyright © 2017");
+		versionNo.setFont(Font.font("Ariel", 12));
+		versionNo.setTextFill(Color.GRAY);
+		versionNo.setTextAlignment(TextAlignment.CENTER);
+
+		Label details = new Label("\"The best study planner since the Gantt Diagram\"");
+		details.setFont(Font.font("Ariel", FontPosture.ITALIC , 24));
+
+		Label stayOnTrack = new Label("Stay on track.");
+		stayOnTrack.setFont(Font.font("Ariel", FontWeight.BOLD, 18));
+
+		Label summary = new Label("Life gets complicated, and keeping track of homework and exam\n"
+				+ "dates is not the first thing on your mind. RaiderPlanner will be "
+				+ "there to help you out.");
+		summary.setFont(Font.font("Ariel", 18));
+		summary.setTextAlignment(TextAlignment.CENTER);
+
+		Label credits = new Label("Created By:\n"
+				+ "Roberto Sánchez\n"
```

Comment:
I don't think my name should be listed here specifically. I like the class credit and the based-on part.